webview/src/experiments/components/table/RowContextMenu.tsx

```diff
@@ -78,15 +86,15 @@ const getMultiSelectMenuOptions = (
     experimentMenuOption(
       selectedIds,
       'Plot',
-      MessageFromWebviewType.SET_EXPERIMENTS_FOR_PLOTS,
+      MessageFromWebviewType.SET_EXPERIMENTS_AND_OPEN_PLOTS,
```

Comment:
[I] Following on from my previous comment about the 'Plot' action for a single experiment I think this should also be additive. If there are 3 experiments already selected then the multi-select option should add to those three experiments (not replace them) and be available up until it will select > 7 experiments. At which point it should be greyed out.